services/provider/server/storageclaim.go

```diff
@@ -121,3 +126,19 @@ func (s *storageClassClaimManager) Delete(ctx context.Context, consumerUUID, sto
 
 	return nil
 }
+
+// Get returns the storageClassClaim resource using storageClassClaimName
+// and consumerUUID.
+func (s *storageClassClaimManager) Get(ctx context.Context, consumerUUID, storageClassClaimName string) (*ocsv1alpha1.StorageClassClaim, error) {
+	generatedClaimName := getStorageClassClaimName(consumerUUID, storageClassClaimName)
+	storageClassClaimObj := &ocsv1alpha1.StorageClassClaim{}
+	err := s.client.Get(ctx, types.NamespacedName{Name: generatedClaimName}, storageClassClaimObj)
```

Comment:
Claim is a namespace scoped object, you should pass the namespace name to the get call